### DISCUSSION TITLE
install-haskell ubuntu 2404 failed with connection reset by peer

### DISCUSSION THREAD

**Discussion by zoezhangmattr** [actions/virtual-environments#10748] [Installed software]
try to use the runner images repo to build ubuntu 24 image, master branch, but somehow install haskell always fail, not sure whats going on, 
ubuntu24.04 run install-haskell.sh failed
Provisioning with shell script: scripts/build/install-haskell.sh
   
curl: (56) Recv failure: Connection reset by peer
ghcup: command not found

**Comment by zoezhangmattr** [comment]
seems like the server issue, might be just server issue then

**Reply by zoezhangmattr** [reply]
https://github.com/haskell/ghcup-hs/issues/1135

**Comment by lakshminarayana02** [comment]
HI
This type of error is typically caused by local network issues ,  please check and try again
